src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/ManageService.cs

```diff
@@ -75,19 +75,9 @@ public async Task<IdentityResult> RemoveLoginAsync(ApplicationUser user, string
             return await this._userManager.RemoveLoginAsync(user, loginProvider, providerKey).ConfigureAwait(false);
         }
 
-        public AuthenticationProperties ConfigureExternalAuthenticationProperties(string redirectUrl, string userId)
-        {
-            throw new NotImplementedException();
-        }
-
         public AuthenticationProperties ConfigureExternalAuthenticationProperties(string provider, Uri redirectUrl, string userId)
         {
             throw new NotImplementedException();
```

Comment:
there is still NotImplementedException :(

Reply:
Deleted already :)